Deploy/pack_poseEstimation_run.spec

```diff
@@ -0,0 +1,69 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+from PyInstaller.utils.hooks import collect_all
+from PyInstaller.utils.hooks import copy_metadata
+from PyInstaller.utils.hooks import collect_dynamic_libs
+
+
+block_cipher = None
+
+datas           = []
+binaries        = []
+hiddenimports   = []
+excludes        = []
+
+# datas += copy_metadata('deeplabcut', recursive=True)
+tmp_ret = collect_all('deeplabcut')
+datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
+
+excludes = ["tensorflow","PySide6", "PyQT6", "PyQT5", "IPython", "Markdown", "jupyter"]
```

Comment:
Please see if matplotlib can be excluded

Reply:
Excluded. 
For matplotlib I had to use mock objects for submodules (it does not look neat because it's a long list of submodules  but it does the job.)